code/script.js

```diff
@@ -1,13 +1,85 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat")
+const nameForm = document.getElementById("name-form")
+const inputwrapper = document.getElementById("input-wrapper")
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
+const nameInput = document.getElementById("name-input")
+
+const handleNameInput = (event) => {
+  event.preventDefault()
+
+  showMessage(nameInput.value, "user")
+
+  nameInput.value = ""
+
+  setTimeout(()=> question1(), 1000)
+
+  
+
+  }
+
+  const question1 = () => {
+    showMessage("Hur ofta vill du spara?", "bot")
+    inputwrapper.innerHTML = `
+    <button id="button1">Lite varje månad</button>
+    <button id="button2">Ett engångsbelopp</button>`
```

Comment:
Nice buttons 👌 To follow conventions and improve readability, you could indent your content like this:
```
inputwrapper.innerHTML = `
  <button id="button1">Lite varje månad</button>
  <button id="button2">Ett engångsbelopp</button>
`
```
If you want to learn more about conventions - take a look at [our code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e).

Reply:
Maybe the ids could be `monthly` and `oneTime` or similar, to be more descriptive?